lib/iris/fileformats/netcdf.py

```diff
@@ -977,6 +977,11 @@ def __setitem__(self, keys, arr):
         self.target[keys] = arr
 
 
+# NOTE : this matches :class:`iris.experimental.ugrid.Mesh.LOCATIONS`,
+# but in a specific preferred order.
```

Comment:
@pp-mo Begs the obvious question... what's the reasoning behind the specific order?

If a developer was to change the order, what would it mean?

Reply:
It really just controls which order the coords + connectivities appear in the file.
Let me try fixing the comment to make that clear...

Reply:
```suggestion
# but in the preferred order for coord/connectivity variables in the file.
```

Reply:
? ok ?